js/wizard/steps/booking-confirmation-step/booking-confirmation-step.js

```diff
@@ -38,6 +38,9 @@ export const bookingConfirmationStep = (props) => {
 
    //I had to make displaying this message dependant on an if, because I'd get error
    //due to trying to call .get() on undefined.
+   //    If I have a variable that can be either `present` (object, basic type, etc.
```

Comment:
**suggestion:** Preferably just check it by throwing an error before. It will be much better user experience that just a `return`, because user will know what happened